_episodes/02-lists.md

```diff
@@ -20,11 +20,26 @@ list[2:9]), in the same way as strings and arrays."
 - "Strings are immutable (i.e., the characters in them cannot be changed)."
 ---
 
-Just as a `for` loop is a way to do operations many times,
-a list is a way to store many values.
+In the previous episode, we analyzed only one file. We still have eleven more 
+to be analyzed...
+
+Our goal with this Python lesson is to automate our analysis.
+Simply, we will write one code that will analyze all files,
+instead of changing the file name and running the code eleven more times. 
```

Comment:
I think this is great how the episode has been changed here to reference the overall goal of the lesson to analyze all the inflammation data in an automated fashion. I think the phrase "one code" could be confusing and the wording could benefit from specifying that the files are the inflammation data files. Also, I think the "automated code" phrase may be used more times than needed here. @ngrafgr and @maxim-belkin sorry, I am about to board a plane and will have limited Internet access the next couple days, so please don't wait on me for further review here. I think the design of these changes are good--just needs a bit of word tweaking, so perhaps @maxim-belkin can make further suggestions.